install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -109,6 +109,8 @@ data:
       {{- end }}
         - --proxyAdminPort
         - {{ "[[ .ProxyConfig.ProxyAdminPort ]]" }}
+        - --concurrency
+        - {{ "[[ .ProxyConfig.Concurrency ]]" }}
```

Comment:
What is the default value ? What happen if 0 is passed ? 

Don't we need something in values.yaml ( matching the current default value, and with comments ) ?

Reply:
Also: any updates to sidecar injector must also be matched in istio-remote

Reply:
Oh, the other way around then ? Did someone already add it to istio/ ?

Reply:
Nope, I made the changes to both files.